pkg/dinghyfile/builder.go

```diff
@@ -182,6 +185,33 @@ func (b *PipelineBuilder) ProcessDinghyfile(org, repo, path, branch string) erro
 	return nil
 }
 
+func unwrapFront50Error(err error) error {
+	// Front50/OPA errors are in the form {"error": "BadRequest", "message": "foo"}
+	// So let's attempt to destructure that value and report it downstream so
+	// users have a nicer error message. If we can't parse the message for
+	// whatever reason, then we'll just end up returning the previous error instead.
+	var fr *plank.FailedResponse
+	if errors.As(err, &fr) {
```

Comment:
Make sure the Docker image is updated to use a version of Go that supports `errors.As`. This is a relatively new feature that isn't available in older versions of Go.

Reply:
Yep! [Dinghy's Docker file is currently on 1.13](https://github.com/armory-io/dinghy/blob/bb15163eb1589542fcae8b5d8fba1d31a1aae7bc/Dockerfile#L1), thanks for lookin' out!